playbooks/roles/cluster/files/LE_privatekey_template

```diff
@@ -0,0 +1 @@
+{{ le_key }}
```

Comment:
Should this be a template rather than a file?

Reply:
It could be, but then I'd have to write it out locally, doing it this way means it all stays in memory and the key is never written to a file. I thought it was cleaner and more secure but I'm not against doing it the normal template way.

Reply:
Good to know, I didn't realise files could be handled this way. Always thought they had to be created as a file locally and passed in 🤦‍♂️